src/iperf_error.c

```diff
@@ -260,11 +260,11 @@ iperf_strerror(int int_errno)
             snprintf(errstr, len, "received an unknown control message");
             break;
         case IESENDMESSAGE:
-            snprintf(errstr, len, "unable to send control message");
+            snprintf(errstr, len, "unable to send control message - port may not be available, the other side may stopped running, etc.");
             perr = 1;
             break;
         case IERECVMESSAGE:
-            snprintf(errstr, len, "unable to receive control message");
+            snprintf(errstr, len, "unable to receive control message - port may not be available, the other side may stopped running, etc.");
```

Comment:
"...the other side may have stopped running..." (add the word "may")

Reply:
Done